src/components/fileDropArea/utils/getValidationErrorMessage.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FileValidationError, FileValidationMessages } from '../types';
+
+export const getValidationErrorMessage = (
+  errors: FileValidationError[],
+  messages: FileValidationMessages,
+) => {
+  if (!errors.length) {
+    return undefined;
```

Comment:
```suggestion
    return;
```
Or why undefined not null?